style/forms/ToggleButton.scss

```diff
@@ -0,0 +1,43 @@
+.ds-form-control-toggle {
+  position: relative;
+  align-items: center;
+  display: inline-flex;
+
+  input {
+    display: none;
+  }
+
+  &__slider {
+    position: relative;
+    overflow: hidden;
+    // float: left;
+    border-radius: 37px;
```

Comment:
I'd consider moving this number into a separate variable (we use it in a few places)

Reply:
Now we use this only in one place.